src/models/constants/TechnologyTypes.js

```diff
@@ -1,22 +1,152 @@
 export const TechnologyTypes = Object.freeze([
-    "React",
+    ".NET",
+    ".NET Core",
+    ".NET Entity Framework",
+    "Abap",
+    "Ada",
+    "Alpine",
+    "Android",
     "Angular",
-    "Vue",
-    "Node.js",
-    "Java",
-    "C++",
+    "Ansible",
+    "Apache",
+    "Apollo",
+    "Appium",
+    "ASP.NET",
+    "ASP.NET Core",
+    "Aurelia",
+    "AWS",
+    "Azure",
+    "Beego",
+    "Browserstack",
     "C",
     "C#",
+    "C++",
+    "Cassandra",
+    "CircleCI",
     "Clojure",
+    "Cobol",
+    "CodeIgniter",
+    "CouchDB",
+    "CSS",
+    "Cypress",
+    "D",
+    "Dart",
+    "DevOps",
+    "Django",
+    "Docker",
+    "Drupal",
+    "Elasticsearch",
+    "Electron",
+    "Elixir",
+    "EmberJS",
+    "Erlang",
+    "ExpressJS",
+    "F#",
+    "FastAPI",
+    "Flask",
+    "Flutter",
+    "Fortran",
+    "Gatsby",
+    "Git",
     "Go",
+    "Google Cloud",
+    "Grafana",
+    "Graphite",
+    "GraphQL",
+    "Groovy",
+    "GTK+",
+    "H2",
+    "Hadoop",
     "Haskell",
-    "Spring Boot",
-    "Android",
-    "Flutter",
-    "Dart",
+    "HTML",
+    "IBM DB2",
+    "iOS",
+    "Iris",
+    "Java",
+    "JavaScript",
+    "Jenkins",
+    "Jest",
+    "Jira",
+    "JQuery",
+    "Julia",
+    "Junit",
+    "Keras",
+    "Koa",
+    "Kotlin",
+    "Kubernetes",
+    "Laravel",
+    "Linux",
+    "LitElement",
+    "Lua",
+    "MacOS",
+    "MariaDB",
+    "Matlab",
+    "Meteor",
+    "Mocha",
+    "MongoDB",
+    "MySQL",
+    "NextJS",
+    "Nginx",
+    "NodeJS",
+    "NoSQL",
+    "Objective-C",
+    "OpenAI",
+    "OpenCL",
+    "OpenCV",
+    "OpenGL",
+    "Oracle Database",
+    "Pascal",
+    "Perl",
+    "Phoenix",
     "PHP",
-    "CSS",
-    "Other",
+    "PHPUnit",
+    "Polymer",
+    "PostgreSQL",
+    "Preact",
+    "Prolog",
+    "Prometheus",
+    "Pytest",
+    "Python",
+    "PyTorch",
+    "Qt",
+    "R",
+    "RabbitMQ",
+    "React",
+    "React Native",
+    "Redis",
+    "Robot Framework",
+    "Ruby",
+    "Rust",
+    "Scala",
+    "Scheme",
+    "Selenium",
+    "Shell",
+    "Spring",
+    "Spring Boot",
+    "SQL",
+    "SQL Server",
+    "Stimulus",
+    "Strapi",
+    "Svelte",
+    "Swift",
+    "Swing",
+    "Symfony",
+    "Tensorflow",
+    "Terraform",
+    "TypeScript",
+    "Unity",
+    "Unreal Engine",
+    "VBA",
+    "VirtualBox",
+    "Visual Basic",
+    "VMWare",
+    "VueJS",
+    "WebdriverIO",
+    "Windows",
+    "WordPress",
+    "Xamarin",
+    "Yii",
+    "Other"
```

Comment:
Add Solidity, Vyper, Simplicity, Rholang

Reply:
Ruby on rails

Reply:
Gastsby

Reply:
Gatsby is already on the list. I added the others